packages/gatsby-plugin-catch-links/README.md

```diff
@@ -1,24 +1,33 @@
 # gatsby-plugin-catch-links
 
-Intercepts local links from markdown and other non-react pages and does a
-client-side pushState to avoid the browser having to refresh the page.
+This plugin intercepts all local links that have not been created in React using [`gatsby-link`](https://gatsbyjs.org/docs/gatsby-link), and replaces their behaviour with that of the `gatsby-link` [`navigate`](https://gatsbyjs.org/docs/gatsby-link/#programmatic-navigation). This avoids the browser having to refresh the whole page when navigating between local pages, preserving the Single Page Application (SPA) feel.
 
-For instance, in a markdown file with relative links (transformed
-to `a` tags by
-[`gatsby-transformer-remark`](/packages/gatsby-transformer-remark/)), this
-plugin replaces the default link behaviour
-with that of [`gatsby-link`'s `navigate`](https://gatsbyjs.org/docs/gatsby-link/#programmatic-navigation), preserving the
-SPA-like page change without reload.
+Example use cases:
 
-Check out the [_Using Remark_ example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-remark) to see this plugin in action.
+- A markdown file with relative links (transformed
+  to `a` tags by
+  [`gatsby-transformer-remark`](/packages/gatsby-transformer-remark/))
+- An `a` tag that has been created by a Content Management System (CMS) WYSIWYG editor
 
-## Install
+## Installation
 
-`npm install --save gatsby-plugin-catch-links`
+```
+npm install --save gatsby-plugin-catch-links
+```
+
+or
+
+```
+yarn add gatsby-plugin-catch-links
+```
```

Comment:
we don't use `yarn` in our docs for users (just convention - we just think that if someone is using `yarn`, they will be able to translate `npm install` into `yarn`, so let's just remove "or" part

Reply:
@pieh no problem, just pushed another commit to fix this.